microsetta_public_api/api/diversity/alpha.py

```diff
@@ -6,16 +6,42 @@
 def get_alpha(sample_id, alpha_metric):
     alpha_repo = AlphaRepo()
     if not all(alpha_repo.exists([sample_id])):
-        return jsonify(error=404, text="Sample ID not found."), 404
+        return jsonify(error=404, text="Sample ID not found."),\
+               404
     alpha_series = alpha_repo.get_alpha_diversity([sample_id],
                                                   alpha_metric)
     alpha_ = Alpha(alpha_series)
     alpha_data = alpha_.get_group_raw().to_dict()
     ret_val = {
         'sample_id': sample_id,
         'alpha_metric': alpha_data['alpha_metric'],
-        'data': alpha_data['data'][sample_id],
+        'data': alpha_data['alpha_diversity'][sample_id],
 
     }
 
     return jsonify(ret_val), 200
+
+
+def alpha_group(body, alpha_metric):
+    sample_ids = body['sample_ids']
+
+    alpha_repo = AlphaRepo()
+    exists = zip(sample_ids, alpha_repo.exists(sample_ids))
```

Comment:
I was a little confused by what was happening here... would this work?

```python
missing = [id_ for id_ in sample_ids if not alpha_repo.exists(id_)]
if len(missing) > 0:
    return jsonify(missing_ids=missing, ...)
```

Reply:
How do you feel about this? https://github.com/biocore/microsetta-public-api/pull/5/commits/2bc42c61d60715b97fbbb0ebab47a942d5c72a45

Reply:
Seems more complex -- the `zip` I don't think is needed. But, I'll defer to the style you prefer here

Reply:
The original thought was that it might be better to make a single call to `exists` than many. Just as happy to kick that down the road a bit. Changed in https://github.com/biocore/microsetta-public-api/pull/5/commits/feee6d026b503ca22fcb9f32c0300d7de14383cf along with the requisite testing, and some additional edge case coverage.

Reply:
Ah. if that's a bottle neck we can optimize later. better to be expressive right now